parity/lib.rs

```diff
@@ -84,7 +84,7 @@ extern crate log as rlog;
 extern crate ethcore_accounts as accounts;
 
 #[cfg(feature = "secretstore")]
-extern crate ethcore_secretstore;
+extern crate parity_secretstore;
```

Comment:
I'd probably have renamed the import here: `extern crate parity_secretstore as secret_store;`, but no big deal.